.github/workflows/docker.yaml

```diff
@@ -53,7 +41,12 @@ jobs:
           # Our Docker tag name should be our branch name with just alphanums
           BRANCH_TAG="$(echo $GHA_REF | sed -e 's:refs/heads/::; s:/:-:g' | tr -cd '[:alnum:]_-')"
           VERSION_TAG=$(./print_version.sh)
-          docker build -t current-build -f docker/Dockerfile.release-linux .
+          docker buildx \
```

Comment:
would it support multi-platform builds for docker images as well? I can update this flow accordingly then.

Reply:
I did that once, and it worked, but it took like 75 minutes, which would be even slower than the Windows build. We might be better off using the cross-compiler for the arm64 Docker image too... can different stages of a multistage Docker build use different architectures?

Reply:
They can, and I implemented it, woo